src/Orleans/Streams/Core/ImplicitConsumerGrainExtensions.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Orleans.Streams
+{
+    public static class ImplicitConsumerGrainExtensions
```

Comment:
do you need to add xml comments here? build your solution and fix warnings if any,